components/aws/sagemaker/batch_transform/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+FROM ubuntu:16.04
+
+RUN apt-get update -y && apt-get install --no-install-recommends -y -q ca-certificates python-dev python-setuptools wget unzip
+
+RUN easy_install pip
+
+RUN pip install boto3==1.9.169 sagemaker pathlib2 pyyaml==3.12
```

Comment:
I thought we always stick to low level api? what's the use case of sagemaker in this case?

Reply:
we wanted users to be able to input built-in algo names instead of the image, so the sagemaker api is being used to get the image URI for built-in algos

Reply:
make sense